modules/__tests__/_bc-Link-test.js

```diff
@@ -1,165 +0,0 @@
-import expect, { spyOn } from 'expect'
-import React, { Component } from 'react'
-import { Simulate } from 'react-addons-test-utils'
-import { render } from 'react-dom'
-import execSteps from './execSteps'
-import createHistory from 'history/lib/createMemoryHistory'
-import Router from '../Router'
-import Route from '../Route'
-import Link from '../Link'
-import shouldWarn from './shouldWarn'
-
-const { click } = Simulate
-
-describe('v1 Link', function () {
-
-  class Hello extends Component {
-    render() {
-      return <div>Hello {this.props.params.name}!</div>
-    }
-  }
-
-  let node
-  beforeEach(function () {
-    node = document.createElement('div')
-  })
-
-  beforeEach(function () {
-    shouldWarn('deprecated')
-  })
-
-  it('knows how to make its href', function () {
-    class LinkWrapper extends Component {
-      render() {
-        return <Link to="/hello/michael" query={{ the: 'query' }} hash="#the-hash">Link</Link>
-      }
-    }
-
-    render((
-      <Router history={createHistory('/')}>
-        <Route path="/" component={LinkWrapper} />
-      </Router>
-    ), node, function () {
-      const a = node.querySelector('a')
-      expect(a.getAttribute('href')).toEqual('/hello/michael?the=query#the-hash')
-    })
-  })
-
-  describe('with params', function () {
-    class App extends Component {
-      render() {
-        return (
-          <div>
-            <Link
-              to="/hello/michael"
-              activeClassName="active"
-            >
-              Michael
-            </Link>
-            <Link
-              to="hello/ryan" query={{ the: 'query' }}
```

Comment:
Need to figure out what to do here... the "link without leading slash" case isn't covered elsewhere in our test suite.